Tasks/WindowsMachineFileCopy/RoboCopyJob.ps1

```diff
@@ -161,16 +161,31 @@ param (
         return ""
     }
     
+    function Get-NetExeCommand
+    {
+        $netExePath = Join-Path -path (get-item env:\windir).value -ChildPath system32\net.exe
```

Comment:
should you use Get-command -Type Application net.exe instead? It will also give you the Source path.  You might however want to check the support of this command and arguments in PS versions. But that might be a better way instead of you having to construct the net.exe path.